src/aurora/gateway/gateway.cc

```diff
@@ -17,23 +17,31 @@
 
 #include <iostream>
 #include <string>
+#include <utility>
 
-#include "nlohmann/json.hpp"
+#define ZLIB_SUFFIX "\x00\x00\xff\xff"
```

Comment:
Please define this macro at the beginning of the `OnMessage` function (where it's actually being used) and undef it at the end of the function.

In the future, a separate wrapper class for handling zlib compression might be introduced.